lib/shopify-cli/helpers/env_file.rb

```diff
@@ -44,21 +45,20 @@ def parse(filename)
         end
       end
 
-      property :app_type, required: true
       property :api_key, required: true
       property :secret, required: true
       property :shop
       property :scopes
       property :host
 
-      def write(ctx, filename)
-        template = self.class.parse_template(app_type.class.env_file)
+      def write(ctx)
+        template = self.class.parse_template(Project.current.app_type.env_file)
         output = []
         template.each do |key, value|
           output << "#{key}=#{send(value)}" if send(value)
         end
         ctx.print_task('writing .env file')
-        ctx.write(filename, output.join("\n") + "\n")
+        ctx.write('.env', output.join("\n") + "\n")
```

Comment:
you’ve got this filename in a couple of spots. worth keeping in a constant?

Reply:
yep!